app/models/manageiq/providers/autosde/storage_manager/host_initiator_group.rb

```diff
@@ -0,0 +1,12 @@
+class ManageIQ::Providers::Autosde::StorageManager::HostInitiatorGroup < ::HostInitiatorGroup
```

Comment:
@gregoryb-at-ibm I just noticed that the `host_initiator_groups` model doesn't have a `:type` column so the subclass won't be set by the persister.

Can you open a PR to manageiq-schema to add a string type column for STI?

Reply:
Will do. It's unrelated to this set of PRs though, right?